docs/read-the-docs/source/developer_guide.rst

```diff
@@ -0,0 +1,324 @@
+Developer guide
+================
+
+Receptor is an open source project that lives at https://github.com/ansible/receptor
+
+Testing
+^^^^^^^
+
+Pull requests must pass a suite of integration tests before being merged into devel.
+
+``make test`` will run the full test suite locally. Some of the tests require access to a Kubernetes cluster, and expects to load in the kubeconfig located at ``$HOME/.kube/config``. One simple way to make these tests work is to start minikube locally before running ``make test``. See https://minikube.sigs.k8s.io/docs/start/ for more information about minikube.
```

Comment:
"Some of the tests require access to a Kubernetes cluster, and expects to load in the kubeconfig located at `$HOME/.kube/config`." 

Reword to:
"Some of the tests require access to a Kubernetes cluster; these tests will load in the `kubeconfig` file located at `$HOME/.kube/config`."

Reply:
good suggestion, thanks